apps/docron/docker-compose.yml

```
@@ -0,0 +1,47 @@
+      traefik.http.routers.docron.tls.certresolver: myresolver
+      # Local domain
+      traefik.http.routers.docron-local-insecure.rule: Host(`docron.${LOCAL_DOMAIN}`)
+      traefik.http.routers.docron-local-insecure.entrypoints: web
+      traefik.http.routers.docron-local-insecure.service: docron
+      traefik.http.routers.docron-local-insecure.middlewares: docron-web-redirect
+      # Local domain secure
+      traefik.http.routers.docron-local.rule: Host(`docron.${LOCAL_DOMAIN}`)
+      traefik.http.routers.docron-local.entrypoints: websecure
+      traefik.http.routers.docron-local.service: docron
+      traefik.http.routers.docron-local.tls: true
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)